frontend/lib/ui/Toolbar/Toolbar.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import { Story } from "@ladle/react";
 
 import { IconPlus } from "@kiesraad/icon";
+import { Button } from "@kiesraad/ui";
 
-import { Button } from "../Button/Button";
 import { Toolbar, ToolbarSection } from "./Toolbar";
```

Comment:
But here we use a relative import within the module.

What do we prefer? I have a small preference for keeping imports relative within the same module.

Reply:
Jetbrains always recommends shortening it to importing from `"@kiesraad/ui"`. Is there any way we can lint this so it's consistent everywhere? I personally like the shorter imports instead of relative.

Reply:
I'll do some research, for now we can leave this as is.